### ISSUE TITLE
trailing_comma incorrectly flags [SomeType]()

### ISSUE THREAD

**Issue by ZevEisenberg** [realm/SwiftLint#911]
This code:

```swift
var imageViews = [UIImageView]()
```

is causing a warning when I have this in my SwiftLint configuration:

```yml
trailing_comma:
  mandatory_comma: true
```

This is with SwiftLint 0.13.1.

**Comment by jpsim** [comment]
Let's try to minimize the false positives and make sure the following doesn't trigger with `mandatory_comma`:

```swift
[Void]()
[(Void, Void)]()
```

Perhaps we can kill two birds with one stone by not reporting violations for single-line collections as reported in #910.

**Comment by jpsim** [comment]
Fixed by @marcelofabri in #912.
